src/afl-fuzz-one.c

```diff
@@ -4339,7 +4351,8 @@ static u8 mopt_common_fuzzing(afl_state_t *afl, MOpt_globals_t MOpt_globals) {
 
               if (temp_len + extra_len >= MAX_FILE) break;
 
-              out_buf = ck_maybe_grow(BUF_PARAMS(out), temp_len + extra_len);
+              out_buf = maybe_grow(BUF_PARAMS(out), temp_len + extra_len);
+              if (unlikely(!out_buf)) { PFATAL("alloc"); }
```

Comment:
if we move the PFATAL into the .h it cannot be forgotten and less code lines overall

Reply:
For multithreaded fuzzing we won't be able to use PFATAL anymore, so I suggest slowly reworking the error handling

Reply:
ah good point, yes then FATAL.
however we have a disadvantage. we only need a check if a realloc was done. however the caller doesnt know if one was done or not and therefore always has to check. but whelp I guess that is the disadvanted of threading

Reply:
As maybe_grow is inlined the compiler can figure that out, but I can double check